themes/barbuk/README.md

```diff
@@ -79,6 +79,15 @@ EOF
 chmod 400 /etc/sudoers.d/keepenv
 ```
 
+## Command duration
+
+Command duration require the the cmd-duration plugin.
+You can enable it with:
+
+```bash
+bash-it enable plugin cmd-duration
```

Comment:
remove this part as it is no longer relevant  
You can instead add docs about the new theme duration feature you have added!

Reply:
Documentation was updated to reflect changes in c2a392a7dfe06b7ae866481744a2291c9a8f1da5